client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -425,16 +429,19 @@ class RequiredPluginsInstallView extends Component {
 		}
 
 		return (
-			<div className="card dashboard__setup-wrapper">
-				{ site && <QueryJetpackPlugins siteIds={ [ site.ID ] } /> }
-				<SetupHeader
-					imageSource={ '/calypso/images/extensions/woocommerce/woocommerce-store-creation.svg' }
-					imageWidth={ 160 }
-					title={ translate( 'Building your store' ) }
-					subtitle={ translate( "Give us a minute and we'll move right along." ) }
-				>
-					<ProgressBar value={ progress } total={ totalSeconds } isPulsing />
-				</SetupHeader>
+			<div className="dashboard__setup-wrapper">
+				<SetupNotices />
+				<div className="card dashboard__plugins-install-view">
+					{ site && <QueryJetpackPlugins siteIds={ [ site.ID ] } /> }
+					<SetupHeader
+						imageSource={ '/calypso/images/extensions/woocommerce/woocommerce-store-creation.svg' }
+						imageWidth={ 160 }
+						title={ translate( 'Building your store' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Get started building your new:' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).